dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/avantis/avantis_schema.yml

```diff
@@ -0,0 +1,76 @@
+version: 2
+
+models:
+  - name: avantis_v1_base_perpetual_trades
+    meta:
+      blockchain: base
+      sector: perpetual
+      contributors: princi
+    config:
+      tags: ['base', 'perpetuals', 'perps', 'avantis','cross-chain']
+    description: >
+      Perpetual swaps/trades table on avantis protocol across blockchains
+    columns:
+      - &blockchain
```

Comment:
We need to add data-tests, including uniqueness test and `check_perpetuals_seed` test.